src/four_gray.toit

```diff
@@ -25,9 +25,6 @@ BLACK ::= 3
 //   1    0   Dark gray
```

Comment:
Is this comment still correct? There is no "canvas" after this comment section anymore.